src/shadowbox/server/manager_service.ts

```diff
@@ -38,12 +39,13 @@ function accessKeyToJson(accessKey: AccessKey) {
     password: accessKey.proxyParams.password,
     port: accessKey.proxyParams.portNumber,
     method: accessKey.proxyParams.encryptionMethod,
+    dataLimit: accessKey.dataLimit,
     accessUrl: SIP002_URI.stringify(makeConfig({
       host: accessKey.proxyParams.hostname,
       port: accessKey.proxyParams.portNumber,
       method: accessKey.proxyParams.encryptionMethod,
       password: accessKey.proxyParams.password,
-      outline: 1,
+      outline: 1
```

Comment:
nit: dangling comma

Reply:
That's I think more technically correct JSON.  It's also consistent with, for example, our response in `getServer()`